### DISCUSSION TITLE
Module_22_物件導向簡介中的問題

### DISCUSSION THREAD

**Discussion by 107602505** [1101-GS4719-C/Python-Community-GS4719-C#315] [Q&A] [answered]
我將以下的程式碼放到jupyter lab執行  
![image](https://user-images.githubusercontent.com/90819282/148732220-6c7d8b60-edd7-48cb-bbd5-c644b7a16671.png)
取得屬性name的資料會出現錯誤 ，但是把反白的那段拿掉就可以執行，所以有點不太懂那段程式碼的意義是甚麼?
![image](https://user-images.githubusercontent.com/90819282/148732122-832e78a3-88ed-4c54-af6c-94e899c5a50a.png)

**Comment by chia-kai-chang** [comment, marked as answer]
其實這一張投影片是兩個範例，用下面兩張圖片來說明比較好理解：

![image](https://user-images.githubusercontent.com/1768891/148735340-60dd2529-0004-4038-8270-330026b54691.png)



![image](https://user-images.githubusercontent.com/1768891/148735272-6f79047b-29a0-41ff-a952-3da377a77fd0.png)

**Reply by chia-kai-chang** [reply]
comment test

**Reply by 107602505** [reply]
謝謝老師 我懂了~

**Comment by chia-kai-chang** [comment]
reply test 2

**Reply by chia-kai-chang** [reply]
comment test 2
